modules/order/config/install/views.view.commerce_orders.yml

```diff
@@ -788,6 +788,71 @@ display:
           destination: true
           entity_type: commerce_order
           plugin_id: entity_operations
+        name:
```

Comment:
Had to add it to use in the combined filter, otherwise it would only search by UID